scripts/mbedtls_dev/bignum_mod_raw.py

```diff
@@ -40,6 +43,385 @@ class BignumModRawTarget(test_data_generation.BaseTarget, metaclass=ABCMeta):
 # END MERGE SLOT 4
 
 # BEGIN MERGE SLOT 5
+class BignumModRawOperation(bignum_common.OperationCommon, BignumModRawTarget, metaclass=ABCMeta):
+    #pylint: disable=abstract-method
+    """Common features for bignum mod raw operations."""
+    modulus_input_cases = [
```

Comment:
I strongly feel we should be aiming at keeping a consistent structure between similar `BignumXXXOperation`. A similar member in `BignumCoreOperation` is called `input_values`.